modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc

```diff
@@ -45,16 +46,16 @@ Here's an example version number, including the patch date:
 
 `4.2.2 - 05-04-2020`
 
+The monthly patches only update applications to the latest date patch for the patch version.
+They do not change the patch version number.
+
+
 === Update Schedule 
 
 The schedule for date patch updates is:
 
-* First Monday of the month, MuleSoft releases the patch.
-+
-You can apply the update manually or let MuleSoft apply the patch automatically.
-+
-See <<Update the Runtime Version Manually>>
-* Third Saturday of the month, MuleSoft automatically applies the patch.
+* In the first full week of the month, MuleSoft releases the date patch.
+* In the third week of the month, MuleSoft automatically applies the date patch that was released in the first week of the month.
```

Comment:
Could we please add a line:
Non-Prod applications will be updated during the week and Production applications will be updated during the weekend.